themes/cupper/layouts/_default/baseof.html

```diff
@@ -47,7 +47,16 @@
           <div>
             <div class="intro">
               <a class="logo" href="/" aria-label="{{ .Site.Title }} home page">
-                <img src="{{ "images/logo.svg" | absURL }}" alt="">
+                <img src="{{ "images/logo.svg" | absURL }}" alt=""><span class="logo-text">
+                  <span class="clear"><mark>&#8239;Usable&#8239;</mark>
+                  </span>
+                  <span class="clear"><mark>&#8239;Software&#8239;</mark>
+                  </span>
+                  <span class="clear"><mark>&#8239;Ecosystem&#8239;</mark>
+                  </span>
+                  <span class="clear"><mark>&#8239;Research&#8239;</mark>
```

Comment:
I don't think we really need to use the `<mark>` semantic element here, since the black background's purpose is more a visual design rather than highlighting the text as a note or important part of text. Correct me if I am wrong. Also, I don't think we need to use 4 different spans, we can use a single span and then fix the width of the parent, such that the text always wrap. That would make styling easier.

Reply:
Less spans always sounds good!